packages/uikit/src/hooks/browser/useRecommendations.ts

```diff
@@ -0,0 +1,41 @@
+import { useUserCountry } from '../../state/country';
+import { useQuery } from '@tanstack/react-query';
+import { QueryKey } from '../../libs/queryKey';
+
+export interface CarouselApp extends PromotedApp {
+    poster: string;
+}
+
+export interface PromotedApp {
```

Comment:
Could you please move interface to core package?

the same as `/fiat/methods`
https://github.com/tonkeeper/tonkeeper-web/blob/d87589789038aae1e96bb00a32de78f0879a6919/packages/core/src/tonkeeperApi/tonendpoint.ts#L171

Reply:
done